GAE/src/org/waterforpeople/mapping/dataexport/GraphicalSurveySummaryExporter.java

```diff
@@ -431,7 +439,8 @@ public int compare(String o1, String o2) {
                 wb.setActiveSheet(isFullReport ? wb.getNumberOfSheets()-1 : 0);
                 wb.write(fileOut);
                 fileOut.close();
-
+                
+                setDocumentProperties(fileName, surveyId);
```

Comment:
I think we should also cut this part out. Maybe we can have it in another branch and only if verified that it doesn't cause any performance issues, merge it. For this issue lets only focus on the generating of the text for 16-digit+ numbers.

Reply:
Ok, I just thought the big change in reporting formats would be a good time to start tagging the files.